CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ Take note of the build status of your pull request, only builds that pass will b
 
 ## License
 
-By contributing your code, you agree to license your contribution under the terms of the APLv2: https://github.com/Skyscanner/kms-issuer/blob/master/LICENCE
+By contributing your code, you agree to license your contribution under the terms of the APLv2: [Licence](https://github.com/Skyscanner/kms-issuer/blob/master/LICENCE)
```

Comment:
Inconsistent use of `licence` vs `license`

Reply:
it's correct: https://www.grammarly.com/blog/licence-license/

>  License is both a noun and a verb in the United States.
>  If you live in any other English-speaking country, you will spell it licence when you use it as a noun and license when you use it as a verb.

Reply:
The more you know., thanks.